packages/main/src/plugin/container-registry.ts

```diff
@@ -1757,6 +1736,80 @@ export class ContainerProviderRegistry {
     }
   }
 
+  private async createContainerDockerode(engineId: string, options: ContainerCreateOptions): Promise<{ id: string }> {
+    // need to find the container engine of the container
+    const engine = this.internalProviders.get(engineId);
+    if (!engine) {
+      throw new Error('no engine matching this container');
+    }
+    if (!engine.api) {
+      throw new Error('no running provider for the matching container');
+    }
+
+    // handle EnvFile by adding to Env the other variables
+    if (options.EnvFiles) {
+      const envFiles = options.EnvFiles || [];
+      const envFileContent = await this.getEnvFileParser().parseEnvFiles(envFiles);
+
+      const env = options.Env || [];
+      env.push(...envFileContent);
+      options.Env = env;
+      // remove EnvFiles from options
+      delete options.EnvFiles;
+    }
+
+    const container = await engine.api.createContainer(options);
+    await this.attachToContainer(engine, container, options.Tty, options.OpenStdin);
+    if (options.start === true || options.start === undefined) {
+      await container.start();
+    }
```

Comment:
it looks like there is a difference then between if I submit a pod or not, 

in one case I'll be attaching streams and eventually start it but not using podman (if I say I want to have a pod) ?